srml/support/src/storage/generator.rs

```diff
@@ -513,6 +513,33 @@ macro_rules! __storage_items_internal {
 	type Now = super::Now<T>;
 }*/
 
+#[macro_export]
+macro_rules! __generate_genesis_config {
+	(WITH_GENESIS
+		$traitinstance:ident $traittype:ident
+		$($fieldname:ident : $fieldtype:ty = $fielddefault:expr ;)*
+	) => {
+		#[derive(Serialize, Deserialize)]
+		#[cfg(any(feature = "std", test))]
```

Comment:
I think that can just be `#[cfg(feature = "std")]`. Tests are always run with `std` enabled and if they are not, it would fail anyway.